### PR TITLE
[Don't Merge] JupiterCore core tests

### PR DESCRIPTION
**Pull request by mbarnett**
This isn't ready to merge yet but gimme dat feedback

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,73 +1,43 @@
...
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 3.0'
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+gem 'simple_form'
+gem 'solrizer', github: 'mbarnett/solrizer', branch: 'solrizable_path_types'
+gem 'turbolinks', '~> 5'
```
Comment (murny) [inline Gemfile]: ewww :upside_down_face:

Comment (mbarnett) [inline Gemfile]: (It's actually been very responsive and snappy, idk what to believe anymore)

File: Gemfile [hunk 2]
```diff
@@ -1,73 +1,43 @@
...
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'web-console', '>= 3.3.0'
 end
 
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-#gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+group :test do
+  gem 'simplecov'
+  # Faker added 0.5 seconds to the test suite per call. Haikunator seems much faster for faking strings
+  gem 'haikunator'
```
Comment (murny) [inline Gemfile]: Seen many people use [ffaker](https://github.com/ffaker/ffaker) for this reason. Never heard of haikunator though.

Comment (mbarnett) [inline Gemfile]: I can give it a shot. Haikunator just spits out heroku-style 'sunny-lotus-2922' strings

File: Gemfile [hunk 3]
```diff
@@ -1,73 +1,43 @@
...
-
 gem 'hydra-works'
-
-gem 'simple_form'
-
-gem 'solrizer', git: 'https://github.com/mbarnett/solrizer.git', branch: 'solrizable_path_types'
 gem 'jquery-rails'
-gem 'bootstrap'
+gem 'mysql2', '>= 0.3.18', '< 0.5'
 gem 'popper_js'
```
Comment (murny) [inline Gemfile]: As discussed, this is from bootstrap. Looks like they fixed this in master: https://github.com/twbs/bootstrap-rubygem/commit/8927eff0d7779cedd4fe773a9b8b5b7174d065c6

Maybe put a comment to review this at a later date?

File: Gemfile (hunk 2 above)
Comment (mbarnett) [inline Gemfile]: I tried it both ways and Haikunator was still about a tenth of a second faster ¯\_(ツ)_/¯

**Comment by mbarnett** [comment]
Closing so I can open the real PR

**Pull request closed without merging**